.github/workflows/nightly.yml

```diff
@@ -14,7 +14,7 @@ on:
         description: "MSIM_TEST_NUM (test iterations)"
         type: string
         required: false
-        default: "30"
+        default: "2"
```

Comment:
What is this value exactly?

Reply:
`MSIM_TEST_NUM` - the number of times to repeat each test. Each repetition of a test is done with a different random seed, starting from the value of `MSIM_TEST_SEED` for the first repetition.

So we run those tests 30 times, which is not needed in the nightly IMO.